src/Tribe/Admin/Help_Page.php

```diff
@@ -66,6 +66,32 @@ public function is_current_page() {
 		return in_array( $current_screen->id, $help_pages );
 	}
 
+	/**
+	 * Checks if the current page is the TEC Help one
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_tec_current_help_page() {
```

Comment:
When I started working on this I tried to find if it could be achieved by using a hook and I didn't find any. Like you also said I saw that there was already logic if its events or tickets, so I followed the same approach to do this.

I've applied your suggestion @juanfra.